src/app/features/test-records/create/create-test-records-routing.module.ts

```diff
@@ -47,7 +47,7 @@ const routes: Routes = [
           {
             path: 'selectDefect',
             component: TestRouterOutletComponent,
-            data: { title: 'Select Defect', roles: Roles.TestResultCreateContingency },
+            data: { title: 'Select defect', roles: Roles.TestResultCreateContingency },
```

Comment:
Why has  the casing changed here?

Reply:
I merged those two ticket in one PR as the bug I mentioned before is a tiny issue